teachertool/src/components/ShareLinkInput.tsx

```diff
@@ -42,6 +42,7 @@ export const ShareLinkInput: React.FC<IProps> = () => {
                 icon={icon}
                 onChange={onTextChange}
                 onEnterKey={onEnterKey}
+                onIconClick={onEnterKey}
```

Comment:
Screen reader doesn't seem to be announcing what the button does (it just says "button"). Does setting iconTitle fix that?

Reply:
I can add it, but I don't know how to test screen readers.

Reply:
Simplest way is to open narrator, which should be installed by default on windows, then tab over the element.

Reply:
You could pass iconTitle for the Button's ariaLabel prop.